app/components/course-page/course-stage-step/community-solution-card/content.ts

```diff
@@ -99,6 +99,13 @@ export default class CommunitySolutionCardContentComponent extends Component<Sig
       this.expandedUnchangedFilePaths = [...this.expandedUnchangedFilePaths, filePath];
     }
   }
+
+  @action
+  handlePublishToGithubButtonClick() {
+    if (typeof this.args.onPublishToGithubButtonClick === 'function') {
+      this.args.onPublishToGithubButtonClick();
+    }
```

Comment:
not sure if this is the best way to handle this, but this was what i got to work. the on modifier errors out if we pass in a callback function that's potentially undefined

Reply:
This works! Added a comment on how it could be simpler